hummingbot/connector/exchange/huobi/huobi_order_book_tracker.py

```diff
@@ -57,12 +49,8 @@ def data_source(self, data_source):
 
     def start(self):
         super().start()
-        self._order_book_stream_listener_task = safe_ensure_future(
```

Comment:
Logic already exist in parent class
https://github.com/hummingbot/hummingbot/blob/d3dbc4064fcf66332bc710e40d278f1f853d23a1/hummingbot/core/data_type/order_book_tracker.py#L95